dotCMS/src/main/java/com/dotmarketing/portlets/workflows/business/WorkflowPermissionsHelper.java

```diff
@@ -0,0 +1,298 @@
+package com.dotmarketing.portlets.workflows.business;
+
+import com.dotcms.business.CloseDBIfOpened;
+import com.dotmarketing.business.APILocator;
+import com.dotmarketing.business.PermissionAPI;
+import com.dotmarketing.business.Permissionable;
+import com.dotmarketing.business.Role;
+import com.dotmarketing.business.RoleAPI;
+import com.dotmarketing.business.UserAPI;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.portlets.contentlet.model.Contentlet;
+import com.dotmarketing.portlets.workflows.model.WorkflowAction;
+import com.dotmarketing.util.InodeUtils;
+import com.dotmarketing.util.Logger;
+import com.dotmarketing.util.UtilMethods;
+import com.google.common.collect.ImmutableList;
+import com.liferay.portal.model.User;
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Set;
+
+class WorkflowPermissionsHelper {
```

Comment:
Helper is just for the Resource/Rest layer, for API we used to use more Utils. Probably not a big deal but just keep that in mind

Reply:
Dang!! I am renaming it to WorkflowActionUtils